src/components/common/applicant-info-3.vue

```diff
@@ -395,15 +395,18 @@ export default class ApplicantInfo3 extends Vue {
     }
   }
 
-  mutateApplicant (key, value) {
```

Comment:
Renamed to reduce confusion (and is also now same as other applicant info pages).